raster/r.sun/testsuite/test_rsun.py

```diff
@@ -3,6 +3,97 @@
 from grass.gunittest.gmodules import SimpleModule
 
 
+class TestRSunMode3(TestCase):
```

Comment:
This should be named something like TestRSunMode1Metadata. Mode 1 and 2 refer to computing irradiance vs irradiation, there is no mode 3.

Reply:
Okay got it!